### PR TITLE
Feature dashboard

### PR DESCRIPTION
**Pull request by DaviMatheus**

### CODE REVIEW

File: lib/features/dashboard/presentation/pages/dashboard_page.dart [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+            icon: Icons.chevron_left_rounded,
+          ),
+          onTap: Navigator.of(context).pop,
+        ),
+        titleSpacing: 0,
+        title: Text(
+          S.current.dashboard_title,
+          style: Theme.of(context).textTheme.headlineMedium,
+        ),
+      ),
```
Comment (MauricioMachadoFF) [inline lib/features/dashboard/presentation/pages/dashboard_page.dart]: remove it dash board dont have a back button

File: lib/features/dashboard/presentation/pages/dashboard_page.dart [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: kSpacingXS),
+          child: ListView(
+            children: [
+            Column(
+            children: [
+              const VerticalGap.sm(),
+              DSButton.primary(
+                onPressed: () {},
+                label: S.current.dashboard_section_reports_button_label,
```
Comment (MauricioMachadoFF) [inline lib/features/dashboard/presentation/pages/dashboard_page.dart:66]: add todo comment here

File: lib/features/dashboard/presentation/pages/dashboard_page.dart [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+    return productsEnding;
+  }
+
+  int _getTotalPrice(List<Product> products) {
+    double totalPrice = 0;
+    for (final product in products) {
+      totalPrice += product.price;
+    }
+    return totalPrice.toInt();
+  }
```
Comment (MauricioMachadoFF) [inline lib/features/dashboard/presentation/pages/dashboard_page.dart]: why to int if price is double? Return must be double.

File: lib/features/dashboard/presentation/pages/dashboard_page.dart [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+  int _getHighestPrice(List<Product> products) {
+    double highestPrice = 0;
+
+    for (final product in products) {
+      if (product.price > highestPrice) {
+        highestPrice = product.price;
+      }
+    }
+
+    return highestPrice.toInt();
```
Comment (MauricioMachadoFF) [inline lib/features/dashboard/presentation/pages/dashboard_page.dart]: same here about being int

File: lib/features/dashboard/presentation/pages/dashboard_page.dart [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+  int _getLowestPrice(List<Product> products) {
+  double lowestPrice = double.infinity;
+
+  for (final product in products) {
+    if (product.price < lowestPrice) {
+      lowestPrice = product.price;
+    }
+  }
+
+  return lowestPrice.toInt();
```
Comment (MauricioMachadoFF) [inline lib/features/dashboard/presentation/pages/dashboard_page.dart]: same here about being int

File: lib/features/dashboard/presentation/widgets/dashboard_information.dart [hunk 6]
```diff
@@ -0,0 +1,70 @@
+
+import 'package:design_system/design_system.dart';
+import 'package:flutter/material.dart';
+
+
+class DashboardCard extends StatelessWidget {
+  const DashboardCard({
+    super.key,
+    required this.message,
+    this.isLoading = false,
```
Comment (MauricioMachadoFF) [inline lib/features/dashboard/presentation/widgets/dashboard_information.dart]: this card is the same one as in the home. change the name to a more generic option and use the same one in both of them.

This change. Find the reference in home an migrate to the shared folder. delete DashboardCard. use the shared reference

File: lib/features/dashboard/presentation/widgets/order_history_cards.dart [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+          Text(
+            S.current.order_id(order.id),
+            style: Theme.of(context).textTheme.bodyLarge,
+          ),          const VerticalGap.nano(),
+          Text(
+            S.current.order_value(order.totalOrderValue.toString()),
+          ),
+          const VerticalGap.nano(),
+          const Divider(thickness: 1, color: kDarkPrimary),
+        ],
```
Comment (MauricioMachadoFF) [inline lib/features/dashboard/presentation/widgets/order_history_cards.dart]: add comma after `kDarkPrimary `

File: lib/features/dashboard/presentation/pages/dashboard_page.dart (hunk 3 above)
Comment (DaviMatheus) [reply lib/features/dashboard/presentation/pages/dashboard_page.dart]: é pq a lista de produtos espera um inteiro

**Pull request merged**